src/config/updateChart.js

```diff
@@ -57,7 +57,12 @@ export default (chart, prevProps, nextProps) => {
 
     if (ticksDiffer) {
         updateTicks(chart, nextProps, mergedContract);
+        const before = chart.xAxis[0].getExtremes().min;
         chart.redraw();
```

Comment:
@borisyankov here is the funny part, the `min` value got changed after `redraw` is invoked

Do you have clue? for example, some plugin that monkey patched the `redraw` method?